File: Web/Web/Startup.fs
namespace Web

open FiltrableAlertBlotter
open Microsoft.AspNet.SignalR
open Owin

type Startup() =
    static member ConfigureSignalR(app: IAppBuilder) =
        let cr = CompositionRoot.Instance
        GlobalHost.DependencyResolver.Register(typeof<StockTickerHub>,
                fun () -> new StockTickerHub(cr.WhenStockChanged, cr.StockTicker))
        app.MapSignalR()

File: Web/WebServer/Startup.cs
using Microsoft.Owin;
using Owin;

[assembly: OwinStartup(typeof(Web.Startup), "Configuration")]
namespace Web
{
    public class Startup
    {
        public void Configuration(IAppBuilder app)
        {
            Microsoft.AspNet.SignalR.StockTicker.Startup.ConfigureSignalR(app);
        }
    }
}